grpc.go

```diff
@@ -193,21 +193,25 @@ func updateContainerCpuset(cgroupPath string, newCpuset string, cookies cookie)
 }
 
 func (a *agentGRPC) onlineCPUMem(req *pb.OnlineCPUMemRequest) error {
```

Comment:
@linzichang @bergwolf 
For sure this patch works, but why don't we simply define two different functions for CPU and memory ?
This way, no need for specific flags, we would simply call the appropriate function:
- `onlineCPU`
- `onlineMemory`

Reply:
We did discuss this a bit above (https://github.com/kata-containers/agent/pull/331#issuecomment-415412330) and I agree it would be a lot clearer (but potentially slower as multiple calls might be required).

Reply:
Yes I can see that!
@bergwolf I understand the concern about multiple calls, but I think it's definitely negligible compared to the whole stack.
And about the API compatibility breakage, we better break it now to make it right, as the project is still not completely stable.